src/test/java/net/snowflake/client/jdbc/SnowflakeDriverIT.java

```diff
@@ -2897,7 +2897,9 @@ public void testPutGet() throws Throwable {
 
     Connection connection = null;
     Statement statement = null;
-    List<String> accounts = Arrays.asList(null, "s3testaccount", "azureaccount", "gcpaccount");
+    boolean testS3stageRegionalUrl = false;
```

Comment:
What does this boolean value do? I see it getting set to "true" later but it doesn't seem to be used.

Reply:
To execute the alter statement only once and when using the last s3testaccount